pkg/k8sutil/pv.go

```diff
@@ -78,8 +78,14 @@ func UpdateReclaimPolicy(client *k8s.Clientset, pv *corev1.PersistentVolume) err
 }
 
 func GetVolumeName(pv *corev1.PersistentVolume) string {
-	// Rook creates rbd image with PV name
-	return pv.Name
+	// check if Volume is provisioned by FlexVolume Driver.
```

Comment:
add E2E testing to make sure everything is working fine?

Reply:
Tested the same manually for now, @subhamkrai will add CI for the same in a follow up PR :+1:

Reply:
sounds good to wait. Merge this PR once the CI is added and things are tested in the CI?

Reply:
create PR #40